dotnetv3/S3/ObjectTagExample/ObjectTag.cs

```diff
@@ -75,7 +76,7 @@ public static async Task PutObjectsWithTagsAsync(IAmazonS3 client, string bucket
                     .ForEach(t => Console.WriteLine($"Key: {t.Key}, Value: {t.Value}"));
 
                 // Replace the tagset new tags.
```

Comment:
Review for sense:
'Replace the tagset new tags.'

Reply:
I ended up removing the comment. It's obvious that we're creating a new set of tags here.